README.md

```diff
@@ -253,10 +246,18 @@ env_var = FilterVariable().with_alias('env')\
 .with_is_required(True)\
 .with_value('prod')
 
+aws_src = FilterSource().with_property("aws_region").with_value('us-west-2')
+
+time = FilterTime().with_start("-1h").with_end("Now")
```

Comment:
I'm wondering if it makes sense to provide a constant for this. Do we know if SignalFx treats this value as case-insensitive? Just trying to think of ways to reduce user error here haha.

Reply:
It doesn't allow upper case unit indicators. We are already checking that and raising an error if it's not a lower case as well
```python
                    elif st[-1:] not in ['m', 'h', 'd', 'w']:
                        raise ValueError(
                            "Start time value must end with a units indicator. Allowed units indicators are "
                            "m, h, d, w representing minutes, hours, days, and weeks respectively. Instead, "
                            "got '{0}'".format(st[-1:]))

```

Reply:
Also, start and end times can be either Strings or Integers. So, had to implement this way

Reply:
Makes sense. Does that also apply for `Now` vs `now`?

Reply:
Yes. But, I was converting it to title case and checking which I shouldn't. Fixed now